nixos/doc/manual/release-notes/rl-2311.section.md

```diff
@@ -38,6 +38,8 @@
 
 - `fileSystems.<name>.autoResize` now uses `systemd-growfs` to resize the file system online in stage 2. This means that `f2fs` and `ext2` can no longer be auto resized, while `xfs` and `btrfs` now can be.
 
+- `services.lemmy.settings.federation` was removed in 0.17.0 and no longer has any effect. Enabling federation is now controlled by configuring the instance's hostname.
```

Comment:
Can you link any upstream documentation on this change? How is the instance hostname relevant to federation?